app/controllers/admin/users/list.js

```diff
@@ -16,6 +16,8 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
         extraValuePaths : ['firstName', 'deletedAt'],
         cellComponent   : 'ui-table/cell/admin/users/cell-first-name',
         width           : 155,
+        isSortable      : true,
```

Comment:
why would you sort users by id, that's random and will confuse people.

Reply:
Actually I wanted to sort them by name, I didn't notice this , I will fix and make changes regarding `firstName` and `id`